mesonbuild/backend/ninjabackend.py

```diff
@@ -462,10 +462,8 @@ def detect_vs_dep_prefix(self, tempfilename):
             return open(tempfilename, 'a', encoding='utf-8')
         filename = os.path.join(self.environment.get_scratch_dir(),
                                 'incdetect.c')
-        with open(filename, 'w') as f:
-            f.write('''#include<stdio.h>
-int dummy;
-''')
+        with open(filename, 'w', encoding='utf-8') as f:
+            f.write('#include<stdio.h>\nint dummy;\n')
```

Comment:
Unrelated string change.

Also I think the change looks ugly!

Reply:
textwrap.dedent is best for this I think

Reply:
switched to `dedent`.